openedx/core/djangoapps/commerce/utils.py

```diff
@@ -2,17 +2,190 @@
 
 
 import requests
+import slumber
+import datetime
+import json
+import os
+import socket
 from django.conf import settings
-from edx_rest_api_client.auth import SuppliedJwtAuth
-from edx_rest_api_client.client import EdxRestApiClient
+from edx_rest_api_client.auth import BearerAuth, JwtAuth, SuppliedJwtAuth
+from edx_rest_api_client.__version__ import __version__
```

Comment:
I think we should pull out this value from the edx-rest-api-client library and make it a constant here, since we don't want this value to change as the library changes, since the deprecated client won't be updated as the library changes.